devbox.go

```diff
@@ -156,3 +156,27 @@ func (d *Devbox) saveCfg() error {
 	cfgPath := filepath.Join(d.srcDir, configFilename)
 	return cuecfg.WriteFile(cfgPath, d.cfg)
 }
+
+func (d *Devbox) convertToPlan() (*plansdk.Plan, error) {
+	configStages := []*Stage{d.cfg.InstallStage, d.cfg.BuildStage, d.cfg.StartStage}
+	planStages := []*plansdk.Stage{{}, {}, {}}
+
+	for i, stage := range configStages {
```

Comment:
Why do you marshal and unmarshal from JSON? Is it to "copy" over the fields? I think you should just copy the right fields one by one, but if you're trying to make it more generic, use `mergo` instead?